app/views/devise/invitations/edit.html.erb

```diff
@@ -15,6 +15,12 @@
     </div>
   <% end %>
 
+  <div class=" mb-8">
+    <%= f.label :time_zone, class: 'label label-text' %>
+    <%= f.time_zone_select(:time_zone, ActiveSupport::TimeZone.country_zones('US'), {}, {class:"select select-bordered w-full max-w-xs"}) %>
```

Comment:
We want to allow all country codes. I think there are ways to prioritize some which might be useful?

Reply:
This setting shows all timezones. the second option `ActiveSupport::TimeZone.country_zones('US')` only prioritizes the US time_zones, it places them on top.